src/modules/pages/components/Dashboard/Admin/AddTask/AddTask.jsx

```diff
@@ -107,7 +107,9 @@ const AddTask = ({ colonyClient, network, user, wallet }: Props) => {
       // eslint-disable-next-line no-undef
       fetch(
         // eslint-disable-next-line max-len
-        `${server}/api/contribution?sessionID=${user.session.id}&address=${wallet.address}&network=${network.slug}`,
+        `${server}/api/contribution?sessionID=${user.session.id}&address=${
```

Comment:
If we change to this, we no longer need `eslint-disable-next-line`. My editor still prefers that former. :thinking: